app/reviews.py

```diff
@@ -0,0 +1,21 @@
+from flask import render_template, request, url_for, redirect
+from flask_login import current_user
+
+from .models.review import Review
+
+from flask import Blueprint
+bp = Blueprint('reviews', __name__)
+
+
+@bp.route('/reviews', methods=['GET','POST'])
```

Comment:
Unable to test this with the current Users and Reviews sample data, since I can't log in as flappy penguin. Grab the more detailed Users.csv from main (all passwords = qwerty) and try again?